CKAN.schema

```diff
@@ -144,6 +144,25 @@
             "items"       : { "type" : "string" },
             "uniqueItems" : true
         },
+        "replaced_by" : {
+            "description" : "Optional pointer to mod that should be selected instead and treated as an update to this mod",
+            "type" : "object",
+            "properties" : {
```

Comment:
Could this use `"$ref": "#/definitions/relationship"` like depends, recommends, suggests, supports, and conflicts do? That way we wouldn't have to specify the names and descriptions in multiple places.

Reply:
I specifically didn't use the definitions/relationship, because this is a special case with limited options. There is only one possible replacement, and we do not allow max_version (because it makes no sense)